staging/src/k8s.io/apiserver/pkg/server/options/api_enablement.go

```diff
@@ -51,6 +51,11 @@ func (s *APIEnablementOptions) AddFlags(fs *pflag.FlagSet) {
 		"api/legacy are special keys to control all and legacy api versions respectively.")
 }
 
+// Validate validates RuntimeConfig with genericapiservers' APIRegistrationManagers.
```

Comment:
Validate validates RuntimeConfig with a list of APIRegistrationManagers. Usually this list only has one element, the registry of the apiserver of the process. But in the advanced (and usually not recommended) case of delegated apiservers there can be more. Validate will filter out the known groups of each APIRegistrationManagers. If anything is left over after that, an error is returned.

Reply:
Thanks